administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -22,6 +23,61 @@
 $input = $app->input;
 
 $assoc = JLanguageAssociations::isEnabled();
+
+// print_r($input->post);
+
+$component = $input->post->get('component');
+$language = $input->post->get('ref-language');
+
```

Comment:
hummm
i don't think that any component should be harcoded in the associations list view.

In my view, the associations list view should reuse (somehow) the seleted component/view list views.